charts/unikorn/templates/applications.yaml

```diff
@@ -287,7 +287,7 @@ spec:
     - name: extraArgs.enforce-node-group-min-size
       value: 'true'
     - name: image.tag
-      value: v1.27.2
+      value: v1.30.0
```

Comment:
Why this is here in the first place, I cannot tell you, but if you look up ^^^ what you should be striving to alter is the _chart version_ e.g.

```
[Wed 15 May 09:10:01 BST 2024] simon@symphony ~/src/github.com/unikorn-cloud/client-go helm search repo cluster-autoscaler
NAME                                              	CHART VERSION	APP VERSION	DESCRIPTION                                       
cluster-autoscaler/cluster-autoscaler             	9.37.0       	1.30.0     	Scales Kubernetes worker nodes within autoscali...
```

This will usually upgrade the image version implicitly, so not only do you get the new binary, you get the supporting infrastructure for that binary the chart provides.

So if you follow all the links you get to https://github.com/kubernetes/autoscaler/blob/cluster-autoscaler-chart-9.37.0/charts/cluster-autoscaler/values.yaml#L239 so delete this tag override entirely and upgrade the chart!

Basically what @yankcrime has alluded to with a bit more detail 😸